projects/demo/src/modules/components/multi-select/examples/2/index.ts

```diff
@@ -49,16 +49,18 @@ export class TuiMultiSelectExample2 {
 
     readonly testValue = new FormControl([databaseMockData[0]]);
 
-    onSearchChange(searchQuery: string) {
+    onSearchChange(searchQuery: string | null) {
         this.search$.next(searchQuery);
     }
 
     /**
      * Server request emulation
      */
-    private serverRequest(searchQuery: string): Observable<readonly User[]> {
+    private serverRequest(searchQuery: string | null): Observable<readonly User[]> {
         const result = databaseMockData.filter(user =>
-            user.toString().toLowerCase().includes(searchQuery.toLowerCase()),
+            searchQuery
```

Comment:
Isn't it what `TUI_DEFAULT_MATCHER` does?

Reply:
fixed